velodrome/sql/model.go

```diff
@@ -80,12 +80,12 @@ type Assignee struct {
 
 // Comment is either a pull-request comment or an issue comment.
 type Comment struct {
-	Repository       string `gorm:"primary_key;index:repo_created"`
+	Repository       string `gorm:"primary_key;index:repo_comment_created"`
```

Comment:
Is this safe without a schema migration of some form?

It seems like it probably won't matter as long as nothing pays too much attention…

Reply:
Don't indexes just make querying the table faster? AKA you cannot access them directly.

Reply:
And the code has some attractive sounding `db.AutoMigrate()` after the `Open()` call

https://github.com/kubernetes/test-infra/blob/3dbf7d4c04b94e9ef8951139fa9581ebfdd2c0ef/velodrome/sql/mysql.go#L60

Reply:
Correct — they shouldn't be accessed directly, but if some meta-code tries looking at the indexes it might get upset about the inconsistent state between declaration and reality.

That said it's probably fine and we can revert if not.

Reply:
I wouldn't be surprised if previous versions created the index for Comment and then later replaced the index with one for IssueEvent (or visa versa).

Reply:
(and we never realized because it just resulted in lower performance, which we didn't recognize)

Reply:
It certainly seems unlikely it's actually doing what it should be